ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/services/ApiSubmissionServiceSpec.scala

```diff
@@ -327,18 +310,127 @@ class ApiSubmissionServiceSpec
         succeed
       }
   }
+
+  behavior of "command deduplication"
+
+  it should "use deduplication if enabled" in {
+    val partyManagementService = mock[IndexPartyManagementService]
+    val writeService = mock[state.WriteService]
+    val indexSubmissionService = mock[IndexSubmissionService]
+    val mockCommandExecutor = mock[CommandExecutor]
```

Comment:
```suggestion
    val partyManagementService = mock[IndexPartyManagementService]
    val writeService = mock[state.WriteService]
    val indexSubmissionService = mock[IndexSubmissionService]
    val mockCommandExecutor = mock[CommandExecutor]
```

I suggest to have them named consistently, i.e. either all with a `mock` prefix (my preference, can make named parameters redundant) or all without.